js/main.js

```diff
@@ -23,4 +31,87 @@ function handleSubmit(event) {
   data.entries.unshift(newObject);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+  $li.prepend(renderTree(newObject));
+  handleClick();
+}
+
+function renderTree(newObject) {
+  var $li = document.createElement('li');
+
+  var $rowOne = document.createElement('div');
+  $rowOne.setAttribute('class', 'row');
+  $li.appendChild($rowOne);
+
+  var $columnHalfOne = document.createElement('div');
+  $columnHalfOne.setAttribute('class', 'column-half');
+  $rowOne.appendChild($columnHalfOne);
+
+  var $img = document.createElement('img');
+  $img.setAttribute('src', newObject.url);
+  $columnHalfOne.appendChild($img);
+
+  var $columnHalfTwo = document.createElement('div');
+  $columnHalfTwo.setAttribute('class', 'column-half');
+  $rowOne.appendChild($columnHalfTwo);
+
+  var $rowTwo = document.createElement('div');
+  $rowTwo.setAttribute('class', 'row');
+  $columnHalfTwo.appendChild($rowTwo);
+
+  var $h3 = document.createElement('h3');
+  $h3.setAttribute('class', 'margin-top-0');
+  $h3.textContent = newObject.title;
+  $rowTwo.appendChild($h3);
+
+  var $rowThree = document.createElement('div');
+  $rowThree.setAttribute('class', 'row');
+  $columnHalfTwo.appendChild($rowThree);
+
+  var $p = document.createElement('p');
+  $p.textContent = newObject.notes;
+  $rowThree.appendChild($p);
+
+  return $li;
+}
+
+function handleLoad(event) {
+  for (var i = 0; i < data.entries.length; i++) {
+    var value = renderTree(data.entries[i]);
+    $li.appendChild(value);
+  }
+}
+
+if (data.view === 'entries') {
+  handleClick();
+} else if (data.view === 'entry-form') {
+  handleNew();
+}
+
+function handleClick(event) {
+  if (data.entries.length === 0) {
+    $noEntries.className = 'row none';
+  } else {
+    $noEntries.className = 'row hidden none';
+  }
+  data.view = 'entries';
+  for (var i = 0; i < $input.length; i++) {
+    var view = $input[i].getAttribute('data-view');
+    if (data.view === view) {
+      $input[i].className = 'input';
+    } else {
+      $input[i].className = 'input hidden';
+    }
+  }
+}
+
+function handleNew(event) {
+  data.view = 'entry-form';
+  for (var i = 0; i < $input.length; i++) {
+    var view = $input[i].getAttribute('data-view');
+    if (data.view === view) {
+      $input[i].className = 'input';
+    } else {
+      $input[i].className = 'input hidden';
+    }
+  }
```

Comment:
You would benefit from making a single function named something like `switchView` that receives an argument of the name of the view to switch to, and the function should show that view and hide all other views. That way, you won't need to make a new function for each new view created.